hls-plugin-api/src/Ide/Types.hs

```diff
@@ -194,6 +197,7 @@ instance Default Config where
     -- , cabalFormattingProvider     = "cabal-fmt"
     -- this string value needs to kept in sync with the value provided in HlsPlugins
     , maxCompletions              = 40
+    , sessionLoading              = SessionLoadSingleComponent
```

Comment:
I do think we should make the _default_ multi-component. We want people to benefit from this without knowing they have to turn it on! The main thing is that they can turn it _off_ if they need to.

Reply:
People have raised the concern that this is basically an untested cabal feature that may contain many hard to understand bugs, so we should disable it by default, and then enable it by default in the next release.